src/test/java/com/networknt/schema/MaximumValidatorPerfTest.java

```diff
@@ -0,0 +1,62 @@
+package com.networknt.schema;
+
+import org.junit.Test;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class MaximumValidatorPerfTest {
+    MaximumValidatorTest test = new MaximumValidatorTest();
+
+    @Test
+    public void testTime() throws InvocationTargetException, IllegalAccessException {
+        test.setUp();
+        String[] testMethodsToBeExecuted = {"testMaximumDoubleValue"};
+        List<Method> testMethods = getTestMethods(testMethodsToBeExecuted);
+        long start = System.currentTimeMillis();
+        executeTests(testMethods, 200000);
```

Comment:
I don't think we should have performance tests be part of the build. Specially when there isn't an assertion. Could this be ignored?

Reply:
you're right. made it ignored.